client/modules/Registration/Registration.js

```diff
@@ -3,33 +3,113 @@ import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 import styles from './Registration.css'
 
+var validator = require('validator');
+
 //Create a component class
+
+function validate(email, password1, password2, mobile, license) {
+    // true means invalid, so our conditions got reversed
+    return {
+      email: !validator.isEmail(email),
+      password1: password1.length === 0,
+      password2: password1 !== password2,
+      mobile: !validator.isMobilePhone(mobile,'en-AU'),
+      license: license.length === 0,
+    };
+  }
+
 export class Registration extends Component {
 
+    constructor() {
+        super();
+        this.state = {
+          email: '',
+          password1: '',
+          password2: '',
+          mobile: '',
+          license: '',
+          
+          everFocusedEmail: false,
+          everFocusedPassword1: false,
+          everFocusedPassword2: false,
+          everFocusedMobile: false,
+          everFocusedLicense: false,
+          inFocus: '',
```

Comment:
are we going to wire these up to something? i cant see them being used

Reply:
they were going to be used. but I will get rid of them in the interim